blog/2020-05-13-native-server-announce.adoc

```diff
@@ -0,0 +1,91 @@
+---
+layout: blog
+title: Infinispan Native Server Image
+date: '2020-05-132T12:00:00.000-00:00'
+author: Ryan Emerson
+tags: [ "docker", "native", "quarkus" ]
+---
+Starting with Infinispan 11, it's now possible to create a natively compiled version of the Infinispan server.
+
+== TL;DR
+We have a new image that contains a natively compiled Infinispan server and has a footprint of only 281MB. Try it now:
```

Comment:
I'd even add the RSS after the first request if possible.

Reply:
Keep in mind that this is heavily dependent upon -Xmx etc.